app/views/houses/_person.html.haml

```diff
@@ -1,7 +1,3 @@
 %li.list__item
   %h2= link_to(person.full_name, person_path(person.graph_id))
-  %ul
-    - person.incumbencies.sort{ |inc_a, inc_b| inc_b.send(:start_date) <=> inc_a.send(:start_date) }.each do |incumbency|
-      %li
-        = "#{l(incumbency.start_date, format: :default)}" unless incumbency.start_date.nil?
-        = incumbency.end_date.nil? ? " to Present" : " to #{l(incumbency.end_date, format: :default)}"
+  %p= person.incumbencies.select(&:current?).empty? ? 'Former Member' : 'Current Member'
```

Comment:
I'm looking at this to see if I could somehow extend my existing statuses method in parliament-ruby.